tests/test_utils/test_config.py

```diff
@@ -532,3 +532,17 @@ def test_deprecation():
         with pytest.warns(UserWarning):
             cfg = Config.fromfile(cfg_file)
         assert cfg.item1 == 'expected'
+
+
+def test_type():
```

Comment:
How about moving the unit test to `test_merge_delete()`?

Reply:
agree

Reply:
Modified